deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java

```diff
@@ -98,20 +119,34 @@ public boolean trigger(CodeGenContext context) throws CodeGenException {
         return false;
     }
 
+    private boolean isJacksonReactiveClientPresent(CodeGenContext context) {
+        return context.applicationModel().getDependencies().stream()
+                .anyMatch(this::isJacksonReactiveClient);
+    }
```

Comment:
Damn, you're right. ``quarkus-rest-client-reactive-jackson`` doesn't define capabilities, whereas ``quarkus-rest-client-jackson`` does. I can well imagine that they just missed that, because it's just a definition in ``quarkus-extension.yaml`` like:
```
  capabilities:
    provides:
    - "io.quarkus.rest.client.reactive.jackson"
```
Maybe it make sense to raise an issue at the Quarkus project. I can do that if you like. 

However, to be backward compatible we should go with analyzing the dependencies, although I'm still wondering why the ``ResolvedDependency`` class comes from a ``Quarkus Maven`` package and whether your code would also work in case a business app would use ``Gradle``.

Reply:
Yes. Please open an issue if you don't mind.
You can also ask about this Maven/Gradle in the issue and link this conversation to contextualize them. In case this is not the best approach, they can suggest a better one.

Reply:
@hbelmiro FYI https://github.com/quarkusio/quarkus/issues/31756#issuecomment-1463651307